packages/core/src/app.ts

```diff
@@ -1889,6 +1890,29 @@ export class App {
         );
     }
 
+    async autoHandleInvites(): Promise<void> {
```

Comment:
This is where the magic happens. Basically whenever an organisation owner unlocks their app, invites will be sent out automatically to all provisioned members.